src/main/java/com/impactupgrade/integration/hubspot/ContactsClient.java

```diff
@@ -44,4 +54,72 @@ public Contact insert(ContactBuilder contactBuilder) throws DuplicateContactExce
       return response.readEntity(Contact.class);
     }
   }
+
+  public Contact update(
+    String email,
+    String first_name,
+    String last_name,
+    String home_phone,
+    String mobile_phone,
+    String work_phone,
+    String preferred_phone,
+    String address,
+    String city,
+    String state,
+    String zip,
+    String country) {
+
+    ContactBuilder cBuilder = new ContactBuilder();
```

Comment:
Instead of building this here, let the caller pass it to you (similar to how `insert` works, above). That gives 2 benefits: 1) callers can determine what info they want to pass along and 2) it reduces this method's argument list.